app/assets/stylesheets/active_admin/print.scss

```diff
@@ -164,62 +131,22 @@ table {
 
 // Show
 .panel {
-  border-bottom: thick solid #ccc;
   margin-bottom: 3em;
   padding-bottom: 2em;
   page-break-inside: avoid;
-
-  &:last-child {
-    border-bottom: none;
-  }
 }
 
-.comments {
-  form {
-    display: none;
-  }
+.active_admin_comment {
+  border-bottom: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.